.npmrc

```diff
@@ -1 +1 @@
-package-lock=false
+package-lock=true
```

Comment:
This isn't necessary. The default is `true`

Reply:
Yeah, I was just making it explicit since it was a departure. I'll remove.